recipe/meta.yaml

```diff
@@ -1,26 +1,26 @@
 {% set name = "radical.utils" %}
-{% set version = "1.48.0" %}
-{% set sha256 = "7cf0e25226f9f087cc3d32a4215e353231ada82b0ae86aa874063cdc94833c98" %}
+{% set version = "1.60.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/radical_utils-{{ version }}.tar.gz
+  sha256: 9d32aded98ba5464ac6fd7efb4769c024342a884aca824b5b70d9ebd8b81a281
 
 build:
+  noarch: python
```

Comment:
I don't see why this couldn't be `noarch: python` but let me know if anyone knows why not.